src/pytest_html/resources/style.css

```diff
@@ -172,15 +172,15 @@ div.video video {
   border-left: 8px solid transparent;
   border-right: 8px solid transparent;
 }
-.sort-icon .inactive {
+.inactive .sort-icon {
```

Comment:
now its like it was before https://github.com/pytest-dev/pytest-html/commit/e00532d9c8a598fb848d16b0ce23665789e3517a#r45034281

Reply:
For someone who is not so familiar with CSS, why does the order matter? @jkowalleck

Reply:
```css
.inactive .sort-icon {}
```

in a nutshell:  this selector matches an HTML element with class `.sort-icon` that is nested in an HTML element with class `.inactive`.
someting like 
```html
<div class=".inactive">
  <div class=".sort-icon">
    vvv
  </div>
</div>
```

Reply:
Got it! So when the order is wrong, it can't find a matching part in the tree?

Reply:
@BeyondEvil 
> So when the order is wrong, it can't find a matching part in the tree?

that's right. 

i had a mistake in the SCSS:
i did forget to put the `&` at the end. which implied an `&` at the beginning. see [the docs](https://sass-lang.com/documentation/style-rules/parent-selector)

Reply:
Oh nice, a parent selector!